### PR TITLE
adds just a small simple example of using a database with kemal.

### LINKED ISSUE #6: Database
Add `crystal-db` instructions

### PR DESCRIPTION
**Pull request by jwoertink**
This addition gives a small example of using a DB with Kemal. Since Kemal is database agnostic, and there's so many different options out there, I wouldn't imagine you'd want an example of everything. I can add an example of PG and SQLite if you want.

Fixes #6

### CODE REVIEW

**Comment by sdogruyol** [comment]
Having a sample of pg and sqlite3 will be great 👍 Let's not mention individual shards like `crystal-mysql` e.g since they're already being used by `crystal-db`. 

WDYT?

**Comment by jwoertink** [comment]
Sure, I can add those in. They should essentially be the same since using crystal-db unifies the interface. I'll remove the driver links, but I still think we should explain that they are needed. crystal-db doesn't work unless you also have a driver shard installed.

**Comment by jwoertink** [comment]
Ok, there's the update. Let me know how that looks or if you think I should change anything else.

File: _docs/database.md [hunk 1]
```diff
@@ -6,10 +6,66 @@ order: 12
...
 - [crystal-db](https://github.com/crystal-lang/crystal-db): Common DB API for Crystal. Supports PostgreSQL, MySQL and SQLite.
 - [crecto](https://github.com/fridgerator/crecto): Database wrapper for Crystal, inspired by Ecto.
 - [topaz](https://github.com/topaz-crystal/topaz): A simple and useful db wrapper.
 
 Please check the relevant repo for samples and more info.
+
+### Using Crystal DB with Kemal
+To use `crystal-db` with Kemal you will need to make sure it's included in your shard as well as the [compatible driver](https://github.com/crystal-lang/crystal-db#crystal-db) (i.e. mysql, sqlite, pg...)
+
+Using Kemal with MySQL
```
Comment (sdogruyol) [inline _docs/database.md]: How about h4 for these sub headers?

File: _docs/database.md [hunk 2]
```diff
@@ -6,10 +6,66 @@ order: 12
...
+```ruby
+require "kemal"
+require "db"
+require "mysql"
+
+# Open a connection to MySQL
```
Comment (sdogruyol) [inline _docs/database.md]: h4

File: _docs/database.md [hunk 3]
```diff
@@ -6,10 +6,66 @@ order: 12
...
+MYSQL = DB.open("mysql://root@localhost:3306/my_db")
+at_exit { MYSQL.close } # Be sure to close the connection
+
+get "/posts/:id" do |env|
+  title = MYSQL.query_one?("SELECT title FROM posts WHERE id = ?", env.params.url["id"], as: {String})
+  "Loaded post #{title}"
+end
+```
+
+Using Kemal with PostgreSQL
```
Comment (sdogruyol) [inline _docs/database.md]: h4

**Comment by jwoertink** [comment]
Updated sub-headers to be <h4>.

**Comment by sdogruyol** [comment]
Thank you 👍

**Pull request merged**